monitor/api/models/models.py

```diff
@@ -45,51 +56,63 @@ def get_test_coverages(self) -> Dict[str, ModelCoverageSchema]:
         return coverages
 
     @staticmethod
-    def _normalize_dbt_model_dict(model: ModelSchema, is_source: bool = False) -> NormalizedModelSchema:
-        model_name = model.name
+    def _normalize_dbt_node_dict(
+        node: Union[ModelSchema, ExposureSchema],
+        type: Literal["model", "source", "exposure"]
+    ) -> Union[NormalizedExposureSchema, NormalizedModelSchema]:
+        node_name = node.name
 
-        owners = model.owners
+        owners = node.owners
         if owners:
             loaded_owners = try_load_json(owners)
             if loaded_owners is not None:
                 owners = loaded_owners
             else:
                 owners = [owners]
 
-        tags = model.tags
+        tags = node.tags
         if tags:
             loaded_tags = try_load_json(tags)
             if loaded_tags is not None:
                 tags = loaded_tags
             else:
                 tags = [tags]
         
-        normalized_model = json.loads(model.json())
-        normalized_model['owners'] = owners
-        normalized_model['tags'] = tags
-        normalized_model['model_name'] = model_name
-        normalized_model['normalized_full_path'] = ModelsAPI._normalize_model_path(
-            model_path=model.full_path,
-            model_package_name=model.package_name,
-            is_source=is_source
+        normalized_node = json.loads(node.json())
+        normalized_node['owners'] = owners
+        normalized_node['tags'] = tags
+        normalized_node['model_name'] = node_name
```

Comment:
model_name is not relevant for exposures right?

Reply:
So everything got a name (exposures as well), the problem is that changing it into node/artifact name will break the UI.
So for now I wanted to leave it like that.
Will add a comment, and will talk with Nimrod.